src/Domain/CreateInitialElement.hpp

```diff
@@ -17,6 +17,9 @@ class ElementId;
 /*!
  * \ingroup ComputationalDomainGroup
  * \brief Creates an initial element of a Block.
+ *
+ * \details This function creates an element at the refinement level and
+ * position specified by the `element_id` within the `block`.
```

Comment:
add `, assuming that all of its neighboring elements are at the same refinement level`

Reply:
I would find this easier to understand if, instead of "assuming X", the documentation was phrased as a statement about the output of the function. I suggest the following replacement: `. The created element will have one neighbor per direction, and each neighbor will be at the same refinement level as the element.` @kidder Thoughts on this rephrasing?

I also suggest clarifying what neighbors will be if the element is at the edge of a Block. (Will there be a neighbor from the other Block? Or no neighbor? I'm not sure.)

Reply:
okay with me; I would say `one neighbor per direction (or zero on an external boundary)`

Reply:
Combined your suggestions into a text, please check the docs